src/types.ts

```diff
@@ -7,14 +7,20 @@ export type Schema = "v1" | "v2" | "planning" | "atf4" | "pipeline";
 
 // This describes the full structure of the GeoJSON we manage. We constrain the
 // default GeoJSON types and specify feature properties.
-export interface Scheme {
+export interface SchemeCollection {
   type: "FeatureCollection";
   features: FeatureUnion[];
   // Foreign members
-  scheme_name?: string;
   authority?: string;
   origin?: string;
+  schemes: { [refrence: string]: SchemeData };
+}
+
+export interface SchemeData {
+  scheme_name: string;
```

Comment:
This was optional previously. Why don't we keep it like that? It'll simplify a bit of the code in getEmptyScheme